zomes/blog/capabilities/main/code/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "code"
+version = "0.1.0"
+authors = ["Nicolas Luck <nicolas@lucksus.eu>"]
+
+[dependencies]
+serde_json = "1.0"
+hdk = { git = "https://github.com/holochain/hdk-rust"}
```

Comment:
missing 'branch' attribute

Reply:
Just like I mentioned in my other PR, git refs assume master branch, so this actually works (since hdk-rust is on master default branch).